### ISSUE TITLE
Fetch on new map not working

### ISSUE THREAD

**Issue by Milkysunshine** [ChaosRecipeEnhancer/ChaosRecipeEnhancer#133]
Entering zones/maps/anything doesn't fetch. I've restarted the app, changed the settings, restarted the overlay, etc.

**Comment by kosace** [comment]
Which language do you play in? Atm it only works in english. I will try to add the other languages too

**Comment by Milkysunshine** [comment]
I play in English

**Comment by kosace** [comment]
do you have the right log file selected? has to be the client.txt

**Comment by Magoo76** [comment]
Hey, I have the exact same issue it seems. I ran 10 maps or so, restarted the app multiple times, reset the settings, restarted the overlay. My game is in English and I have selected the correct path to client.txt

Is there any other way to fetch? my fetch button doesn't work(it's grayed out) and the filter won't update even while manually updating it in game.

First I thought it was because I had some invisible remove only tabs but even swapping them around did nothing. I used the app at the start of Ritual with ease, but when I updated to the 1.2 version I got the fetching sound once and now it seems broken. 

Sorry and thanks in advance, this is a great tool.

Edit : I just tested the 1.1.9 and it works like a charm :)
